timeseries/src/autogluon/timeseries/dataset/ts_dataframe.py

```diff
@@ -731,9 +773,23 @@ def from_pickle(cls, filepath_or_buffer: Any) -> TimeSeriesDataFrame:
         except Exception as err:  # noqa
             raise IOError(f"Could not load pickled data set due to error: {str(err)}")
 
+    @staticmethod
+    def _ensure_index_is_sorted(ts_df: TimeSeriesDataFrame, method_name: Optional[str] = None) -> TimeSeriesDataFrame:
```

Comment:
I generally disagree with passing in `method_name` just for the purposes of logging. It immediately signals to me "index will be sorted differently based on which method is calling" or "this specifies the sorting algorithm for sorting". I don't think we lose anything if the log doesn't feature the method name.